README.md

```diff
@@ -1,4 +1,5 @@
-# 🍸 Open Drinks
+🍸 Open Drinks
+===
```

Comment:
The formatting looks the same, but can you explain this change, please?

Reply:
The change is very minimal, As I removed the # in-front and added === in the next line. There would not be any changing in format but this is GitHub way of doing things.